partials/tabs/faq.html

```diff
@@ -14,6 +14,6 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
+    $('ul.tabs').tabs();
 });
-
```

Comment:
This removal has nothing to do with this issue, so don't do it. We can make it a newcomer issue 👍

Reply:
but here i added tab script line 17 which is needed to fix the bug @li-boxuan